h2o-r/h2o-package/R/frame.R

```diff
@@ -3794,6 +3794,30 @@ h2o.tolower <- function(x) .newExpr("tolower", x)
 h2o.toupper <- function(x) .newExpr("toupper", x)
 
 #'
+#' Searches for matches to argument ‘pattern’ within each element
+#  of a string column.
+#'
+#' This function has similar semantics as R's native grep function
+#' and it supports a subset of its parameters. Default behavior is
+#' to return indices of the elements matching the pattern. Parameter
+#' `output.logical` can be used to return a logical vector indicating
+#' if the element a matches the pattern (1) or not (0).
+#'
+#' @param pattern A character string containing a regular expression.
+#' @param x An H2O frame that wraps a single string column.
+#' @param ignore.case If `TRUE` case is ignored during matching.
+#' @param invert Identify elements that do not match the pattern.
+#' @return H2OFrame holding the matching positions or a logical vector
+#' if `output.logical` is enabled.
+#' @export
+h2o.grep <- function(pattern, x, ignore.case = FALSE, invert = FALSE, output.logical = FALSE) {
```

Comment:
Ah, okay. Was not aware of that. We've tried to ween away from that in H2O, but if it matches R then I believe it can stay as you have it.